chcemvediet/apps/anonymization/admin.py

```diff
@@ -0,0 +1,73 @@
+# vim: expandtab
```

Comment:
*Komentar sa tyka binarneho `anonymization.odt`, ku ktoremu sa nedaju davat komentare*

K `file` property by sa zislo napisat kedy moze byt NULL. Analogicky ako v help texte.

K `name` by sa zislo napisat, ze sa automaticky vypocita pri vytvarani instancie.

K `debug` si si isty, ze "Should be empty"? Skor "May be empty".

Chyba Ti computed `content`

Reply:
*Komentar sa tyka binarneho `anonymization.odt`, ku ktoremu sa nedaju davat komentare*

Sprav ak sekciu "Administration", kde popises ake checky sa robia. Konkretne:
- Clean orphaned local files of deleted attachment normalizations.
- Integrity check for local files of existing attachment normalizations.

Reply:
:ok: